cli/kafka/config.go

```diff
@@ -44,18 +54,73 @@ type ConfigForIDK struct {
 	BatchMaxStaleness time.Duration
 	Timeout           time.Duration
 
-	Table   string
-	IDField string
-	Fields  []idk.RawField
+	Table       string
+	IDField     string
+	PrimaryKeys []string
+	Fields      []idk.RawField
+
+	Encode string
+}
+
+// Generate a Config struct based on a configuration file
+// Default values for Config struct are defined here
+func ConfigStructFromFile(cfgFile string) (cfg Config, err error) {
```

Comment:
I would recommend called this `ConfigFromFile` (the word "Struct" isn't really helpful).